README.md

```diff
@@ -83,6 +83,9 @@ code name
 # 0x14 formerly had the name "sha3", now deprecated
 ```
 
+## Compatibility note
+
+Implementations should make sure keep the string code `sha3` as an alias for `sha3-512` so that `encode('sha3', digest) == encode('sha3-512', digest)`.
```

Comment:
See https://en.m.wikipedia.org/wiki/SHA-3 sections: instances (show
different postfixes) and examples (show totally different hash values for
same input, not just a truncation of same value)

On Wed, Aug 3, 2016 at 16:57 Richard Littauer <notifications@github.com>
wrote:

> In README.md
> <https://github.com/multiformats/multihash/pull/22#discussion_r73419387>:
>
> > @@ -83,6 +83,9 @@ code name
> >  # 0x14 formerly had the name "sha3", now deprecated
> >  ```
> >
> > +## Compatibility note
> > +
> > +Implementations should make sure keep the string code `sha3` as an alias for `sha3-512` so that `encode('sha3', digest) == encode('sha3-512', digest)`.
>
> Unless I misread the spec correctly, they differ mainly in the size of the
> digest. What I'm not sure about is whether or not sha3 should by default
> mean sha3-512, as opposed to one of the other five (like sha3-256).
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/multiformats/multihash/pull/22/files/0638385dd25a1f5b186819b50c9999a4621117b2#r73419387>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAIcoc4EDpDrw_azWOSUeeTT6sg51rL3ks5qcQC4gaJpZM4HLQD1>
> .
>